static-assets/components/cstudio-forms/controls/file-name.js

```diff
@@ -14,6 +14,74 @@
+  );
+  var locales = JSON.parse(
+    '{"bg":{"Й":"Y","Ц":"Ts","Щ":"Sht","Ъ":"A","Ь":"Y","й":"y","ц":"ts","щ":"sht","ъ":"a","ь":"y"},"de":{"Ä":"AE","ä":"ae","Ö":"OE","ö":"oe","Ü":"UE","ü":"ue","ß":"ss","%":"prozent","&":"und","|":"oder","∑":"summe","∞":"unendlich","♥":"liebe"},"es":{"%":"por ciento","&":"y","<":"menor que",">":"mayor que","|":"o","¢":"centavos","£":"libras","¤":"moneda","₣":"francos","∑":"suma","∞":"infinito","♥":"amor"},"fr":{"%":"pourcent","&":"et","<":"plus petit",">":"plus grand","|":"ou","¢":"centime","£":"livre","¤":"devise","₣":"franc","∑":"somme","∞":"infini","♥":"amour"},"pt":{"%":"porcento","&":"e","<":"menor",">":"maior","|":"ou","¢":"centavo","∑":"soma","£":"libra","∞":"infinito","♥":"amor"},"uk":{"И":"Y","и":"y","Й":"Y","й":"y","Ц":"Ts","ц":"ts","Х":"Kh","х":"kh","Щ":"Shch","щ":"shch","Г":"H","г":"h"},"vi":{"Đ":"D","đ":"d"},"da":{"Ø":"OE","ø":"oe","Å":"AA","å":"aa","%":"procent","&":"og","|":"eller","$":"dollar","<":"mindre end",">":"større end"},"nb":{"&":"og","Å":"AA","Æ":"AE","Ø":"OE","å":"aa","æ":"ae","ø":"oe"},"it":{"&":"e"},"nl":{"&":"en"},"sv":{"&":"och","Å":"AA","Ä":"AE","Ö":"OE","å":"aa","ä":"ae","ö":"oe"}}'
+  );
+
+  function replace(string, options) {
+    if (typeof string !== 'string') {
+      throw new Error('slugify: string argument expected');
+    }
+
+    options = typeof options === 'string' ? { replacement: options } : options || {};
+
+    var locale = locales[options.locale] || {};
+
+    var replacement = options.replacement === undefined ? '-' : options.replacement;
+
+    var trim = options.trim === undefined ? true : options.trim;
+
+    var slug = string
+      .normalize()
+      .split('')
+      // replace characters based on charMap
+      .reduce(function (result, ch) {
+        var appendChar = locale[ch] || charMap[ch] || ch;
+        if (appendChar === replacement) {
+          appendChar = ' ';
+        }
+        return (
+          result +
+          appendChar
+            // remove not allowed characters
+            .replace(options.remove || /[^\w\s$*_+~.()'"!\-:@]+/g, '')
+        );
+      }, '');
+
+    if (options.strict) {
+      slug = slug.replace(/[^A-Za-z0-9\s]/g, '');
+    }
+
+    if (trim) {
+      slug = slug.trim();
+    }
+
+    // Replace spaces with replacement character, treating multiple consecutive
+    // spaces as a single space.
+    slug = slug.replace(/\s+/g, replacement);
+
+    if (options.lower) {
+      slug = slug.toLowerCase();
+    }
+
+    return slug;
+  }
+
+  replace.extend = function (customMap) {
+    Object.assign(charMap, customMap);
+  };
+
+  return replace;
+})();
+
+slugify.extend({ '&': '' });
```

Comment:
@roy is this change so that `&` is just removed, instead of being replaced for the word `and`?

Reply:
yes, @avasquez614. in accordance to the specs, & should be removed. we're not giving config options at this stage so it's currently baked behaviour.